CONTRIBUTING.md

```diff
@@ -1,65 +1,251 @@
-
-We are also happy to accept your PRs to the
-[Flutter website](https://github.com/flutter/website/) repo,
-even if it's just to fix a typo.
-
-For ways to get involved in the Flutter community or to learn about us,
-visit the [Flutter community](https://flutter.dev/community) page.
-
-# Ways to contribute
-
-We encourage you to reach out and join the conversation.
-
-An easy way to send feedback is to "thumbs up" issues important to you
-in either the issue tracker for the [Flutter SDK and API docs][issues],
-or the [flutter.dev website][doc-issues].
-
-Other ways you can contribute:
-
-* [Ask HOW-TO questions that can be answered with specific solutions][so]
-* [Live chat with Flutter engineers and users][chat]
-* [Discuss Flutter, best practices, app design, and more on our
-   mailing list][mailinglist]
-* [Report bugs and request features][issues]
-* [Report API docs bugs][issues]
-* [Submit PRs to the Flutter SDK][PRs]
-* [Request docs for flutter.dev][doc-issues]
-* [Submit PRs to flutter.dev][doc-PRs]
-* [Follow us on Twitter: @flutterdev](https://twitter.com/flutterdev/)
-* [Read the Flutter Publication on Medium](https://medium.com/flutter)
-* [Sign up to Future UX Studies on Flutter](https://flutter.dev/research-signup)
-* [Join the subreddit to keep up with the latest in the Flutter
-   community][reddit]
-* [Join the Discord to connect with other developers
-   and discuss Flutter][discord]
-
-Happy Fluttering!
-
-[issues]: https://github.com/flutter/flutter/issues
-[PRs]: https://github.com/flutter/flutter/pulls
-[discord]: https://discord.gg/rflutterdev
-[doc-issues]: https://github.com/flutter/website/issues
-[doc-PRs]: https://github.com/flutter/website/pulls
-[so]: https://stackoverflow.com/tags/flutter
-[mailinglist]: https://groups.google.com/d/forum/flutter-dev
-[chat]: https://github.com/flutter/flutter/wiki/Chat
-[reddit]: https://www.reddit.com/r/FlutterDev
+If you would like to contribute to the Flutter website,
+we’re happy to have your help! Anyone can contribute,
+regardless of your writing experience or how you view yourself as a writer,
+developer, or grammar stickler.
+
+## Contributors
+
+Before submitting a PR,
+[search for an issue][search-issue] that covers your concern.
+If one exists, add `Fixes #<found-issue>` to your new PR.
+If you don't find one and your change is significant,
+consider [creating an issue][create-issue].
+The issue should describe the problem, how you would plan to fix it,
+and should be assigned to yourself.
+
+When you create a new page or perform a major rewrite of an existing page,
+consider drafting it first in Google Docs to simplify collaboration.
+Once your Google doc has been created,
+add a link to it from your new GitHub issue.
+
+### Draft changes
+
+1. [Create a branch][create-branch] for your change.
+   Consider naming it `fix-<issue_number>` to help others find the related issue.
+   The new branch should be based on the `main` branch or a feature
+   or version branch.
```

Comment:
It doesn't make sense that we would tell the reader what to name their branch. Instead, I would much rather we ask them to use the "fixes #number" syntax as it makes issue management easier: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
This is a suggestion, not a mandate. It gives them direction in case they had no other ideas.